google/cloud/forseti/services/inventory/base/resources.py

```diff
@@ -1053,6 +1101,42 @@ def get_iam_policy(self, client=None):
             self.add_warning(err_msg)
             return None
 
+    @cached('org_policy')
+    def get_org_policy(self, client=None):
+        """Gets Organization policy for this organization.
+
+        Args:
+            client (object): GCP API client.
+
+        Returns:
+            dict: Organization Policy.
+        """
+        try:
+            data, _ = client.iter_crm_organization_org_policies(self['name'])
+            return data
+        except (api_errors.ApiExecutionError, ResourceNotSupported) as e:
+            LOGGER.warning('Could not get Org policy: %s', e)
+            self.add_warning(e)
+            return None
+
+    @cached('access_policy')
+    def get_access_policy(self, client=None):
```

Comment:
same as above

Reply:
Moved it under the resource implementation of Resource Manager Access Policy.

Reply:
Same here, this should fall under organization

Reply:
Done